### PR TITLE
add new EMCALPi0CalibSelection task

### PR DESCRIPTION
**Pull request by EwaGlimos**

### CODE REVIEW

File: PWG/EMCAL/EMCALbase/AliEMCALRecoUtils.cxx [hunk 1]
```diff
@@ -505,13 +505,13 @@ Bool_t AliEMCALRecoUtils::CheckCellFiducialRegion(const AliEMCALGeometry* geom,
   Bool_t okrow = kFALSE;
   Bool_t okcol = kFALSE;
 
-  if (iSM < 0 || iphi < 0 || ieta < 0 )
-  {
+  if (iSM < 0 || iphi < 0 || ieta < 0 ) {
```
Comment (gconesab) [inline PWG/EMCAL/EMCALbase/AliEMCALRecoUtils.cxx:508]: I see this kind of change happening often in recent commits to common code, please this is "my" style. I prefer to leave spaces between lines for readability. Do not change if not really necessary, here or elsewhere. This will also easier to get to the point of the commit.

File: PWG/EMCAL/EMCALbase/AliEMCALRecoUtils.cxx [hunk 2]
```diff
@@ -480,17 +481,16 @@ Bool_t AliEMCALRecoUtils::AcceptCalibrateCell(Int_t absID, Int_t bc,
...
+  if (!cluster){
     AliInfo("Cluster pointer null!");
     return kFALSE;
   }
 
   // If the distance to the border is 0 or negative just exit accept all clusters
-  if (cells->GetType()==AliVCaloCells::kEMCALCell && fNCellsFromEMCALBorder <= 0 )
-    return kTRUE;
+  // if (cells->GetType()==AliVCaloCells::kEMCALCell && fNCellsFromEMCALBorder <= 0 ) return kTRUE;
+  if (cells->GetType()==AliVCaloCells::kEMCALCell ) return kTRUE;
```
Comment (gconesab) [inline PWG/EMCAL/EMCALbase/AliEMCALRecoUtils.cxx:493]: Why this change? The point here is if negative this check it is not done.

File: PWGPP/EMCAL/CMakeLists.txt [hunk 3]
```diff
@@ -52,10 +55,11 @@ generate_dictionary("${MODULE}" "${MODULE}LinkDef.h" "${HDRS}" "${incdirs}")
 
 set(ROOT_DEPENDENCIES Core Geom RIO)
 set(ALIROOT_DEPENDENCIES PWGEMCALbase ANALYSIS ANALYSISalice ESDfilter AOD EMCALUtils ESD STEERBase PWGCaloTrackCorrBase)
+set(ALIPHYSICS_DEPENDENCIES EMCALbase PWGCaloTrackCorrBase PWGCaloTrackCorrBase OADB)
```
Comment (gconesab) [inline PWGPP/EMCAL/CMakeLists.txt:58]: Remove PWGCaloTrackCorrBase from ALIROOT_DEPENDENCIES

**Comment by gconesab** [comment]
apart the changes in AliEMCalRecoUtils that I do not agree, and an additional change in CMakeLists.txt,  the rest is fine

File: PWGPP/EMCAL/CMakeLists.txt (hunk 3 above)
Comment (gconesab) [reply PWGPP/EMCAL/CMakeLists.txt:58]: and PWGCaloTrackCorrBase is added twice

**Comment by gconesab** [comment]
+test

**Comment by FriederikeBock** [comment]
+1
